src/plugins/mirror/src/main/java/org/drftpd/mirror/master/MirrorSettings.java

```diff
@@ -110,22 +109,28 @@ public void reload() {
             }
 
             // Handle slaves
-            List<String> slaves = new ArrayList<>(Arrays.asList(cfg.getProperty(id + ".slaves", "").trim().split("\\s")));
-            for (String slaveName : slaves) {
-                try {
-                    GlobalContext.getGlobalContext().getSlaveManager().getRemoteSlave(slaveName);
-                } catch(ObjectNotFoundException e) {
-                    logger.error("Slave with name [{}] does not exist, config error for id {}", slaveName, id, e);
+            List<String> slaves = new ArrayList<>();
+            if (cfg.getProperty(id + ".slaves", null) != null) {
```

Comment:
the `, null` is redundant as it will output null if it cannot find it already.
See https://docs.oracle.com/en/java/javase/15/docs/api/java.base/java/util/Properties.html#getProperty(java.lang.String)